model/finance.php

```diff
@@ -453,6 +481,19 @@ protected function zapoctiZustatek() {
     $this->log('Zůstatek z minulých let', $this->zustatek, self::ZUSTATEK);
   }
 
+  protected function zapoctiStavSleva() {
+    $rok = ROK;
+    $uid = $this->u->id();
+    $o = dbQuery("
```

Comment:
Tu doporučuju dát `dbOneCol`, které ti vyplivne rovnou jednu hledanou hodnotu. Podmínky do where se tam pak nasypou přes parametry dotazu, viz třeba https://github.com/gamecon-cz/gamecon/blob/master/admin/scripts/modules/prava.php#L23 (jak se dívám zrovna v financích jsme to dosud prasili, snad to časem zlepšíme)